state-chain/pallets/cf-ingress-egress/src/boost_pool.rs

```diff
@@ -273,7 +292,7 @@ where
 			amount.saturating_accrue(excess_contributed);
 
 			if let Some(amount) = boosters_to_receive.get_mut(lucky_id) {
-				amount.saturating_accrue(remaining_to_receive)
+				amount.total.saturating_accrue(remaining_to_receive)
```

Comment:
Will this eventually create a discrepancy between the total amount and the fees collected? maybe we should just add the `remaining_to_receive` to the fee here as well?

Reply:
Yeah I think it makes sense to add to the fee here too. Any extra ought to be considered as part of their earned fees.